deployment-template/ec2/data-prepper-ec2-deployment-cfn.yaml

```diff
@@ -70,43 +109,49 @@ Resources:
-              content: !Sub |
-                entry-pipeline:
-                  delay: "100"
-                  source:
-                    otel_trace_source:
-                      ssl: false
-                      health_check_service: true
-                  sink:
-                    - pipeline:
-                        name: "raw-pipeline"
-                    - pipeline:
-                        name: "service-map-pipeline"
-                raw-pipeline:
-                  source:
-                    pipeline:
-                      name: "entry-pipeline"
-                  prepper:
-                    - otel_trace_raw_prepper:
-                  sink:
-                    - elasticsearch:
-                        hosts: [ "${AWSElasticsearchEndpoint}" ]
-                        aws_sigv4: true
-                        aws_region: "${AWSElasticsearchRegion}"
-                        trace_analytics_raw: true
-                service-map-pipeline:
-                  delay: "100"
-                  source:
-                    pipeline:
-                      name: "entry-pipeline"
-                  prepper:
-                    - service_map_stateful:
-                  sink:
-                    - elasticsearch:
-                        hosts: [ "${AWSElasticsearchEndpoint}" ]
-                        aws_sigv4: true
-                        aws_region: "${AWSElasticsearchRegion}"
-                        trace_analytics_service_map: true
+              content: !Sub
+                - |
+                  entry-pipeline:
+                    delay: "100"
+                    source:
+                      otel_trace_source:
+                        ssl: false
+                        health_check_service: true
+                    sink:
+                      - pipeline:
+                          name: "raw-pipeline"
+                      - pipeline:
+                          name: "service-map-pipeline"
+                  raw-pipeline:
+                    source:
+                      pipeline:
+                        name: "entry-pipeline"
+                    prepper:
+                      - otel_trace_raw_prepper:
+                    sink:
+                      - elasticsearch: ${elasticsearchConfig}
+                  service-map-pipeline:
+                    delay: "100"
+                    source:
+                      pipeline:
+                        name: "entry-pipeline"
+                    prepper:
+                      - service_map_stateful:
+                    sink:
+                      - elasticsearch: ${elasticsearchConfig}
+                - elasticsearchConfig: !If
+                    - NoMasterUser
+                    - !Sub "\n
+                    \        hosts: [ \"${AWSElasticsearchEndpoint}\" ]\n
+                    \        aws_sigv4: true\n
+                    \        aws_region: \"${AWSElasticsearchRegion}\"\n
+                    \        trace_analytics_service_map: true"
```

Comment:
Can this be done without the \n's? Something like `!Sub |` similar to how this whole config block is done?

Reply:
I was not able to adjust indentation properly with the literal style as any leading space will be trimmed for new lines. Any ideas?

Reply:
Ah, not a blocker. Was just curious if it could be avoided